src/src/Helpers/_Plugin_Helper_serial.cpp

```diff
@@ -231,6 +241,8 @@ void serialHelper_webformLoad(ESPEasySerialPort port, int rxPinDef, int txPinDef
       options[index] = option;
       ids[index]     = i;
       ++index;
+
+      if (index == NR_ESPEASY_SERIAL_TYPES) { index = 0; } // Restart at begin of list
```

Comment:
This is quite confusing code (as I first stated it would never be true, which is wrong...)
Maybe a short comment where `index` is initialized to why it is set as the last element of the array?